webpack.config.js

```diff
@@ -0,0 +1,22 @@
+var webpack = require('webpack')
+
+module.exports = {
+  entry: './index.js',
+  output: {
+    path: __dirname,
+    filename: 'fuzzyfind.min.js'
+  },
+  plugins: [
+    new webpack.optimize.UglifyJsPlugin({
+      compress: { warnings: false },
+    }),
+    new webpack.BannerPlugin([
```

Comment:
MIT says:
~~~
...
The above copyright notice and this permission notice shall be included in
all copies or substantial portions of the Software.
...
~~~